src/LessMsi.Gui/MainFormPresenter.cs

```diff
@@ -557,12 +561,37 @@ internal void BeginSearching(string searchTerm)
 			    }
 			    else
 			    {
-				    dataSource = this.fileDataSource.Where(x => x.Component.Contains(searchTerm) || x.Directory.Contains(searchTerm) || x.Name.Contains(searchTerm) || x.Version.Contains(searchTerm)).ToList();
+				    dataSource = this.fileDataSource.Where(
+						x => x.Component.Contains(searchTerm, StringComparison.OrdinalIgnoreCase) ||
+						x.Directory.Contains(searchTerm, StringComparison.OrdinalIgnoreCase) ||
+						x.Name.Contains(searchTerm, StringComparison.OrdinalIgnoreCase) ||
+						x.Version.Contains(searchTerm, StringComparison.OrdinalIgnoreCase)
+						).ToList();
 					Status(string.Format("{0} files found.", dataSource.Count));
 			    }
-				ViewLeakedAbstraction.fileGrid.DataSource = dataSource;
+				fileGrid.DataSource = dataSource;
 		    }
-
 	    }
+
+		internal void ExecuteTableSearch(DataGridView _, string searchTerm)

```

Comment:
Lets remove the `DataGridView` arg here. There is no reason for it. in `ExecuteFileSearch` it would be ideal to refactor that so that it was using the IMainFormView too so it wrote `View.SetFileGridDataSource` in the same way this one does.